pootle/apps/pootle_data/abstracts.py

```diff
@@ -77,3 +79,21 @@ class Meta(object):
         blank=False,
         default=0,
         db_index=True)
+
+
+class AbstractPootleChecksData(models.Model):
+
+    class Meta(object):
+        abstract = True
+
+    name = models.CharField(
+        max_length=64,
+        db_index=True)
+    category = models.IntegerField(
+        null=False,
+        default=Category.NO_CATEGORY)
+    count = models.IntegerField(
+        null=False,
+        blank=False,
+        default=0,
+        db_index=True)
```

Comment:
@unho seems there is point http://sqlmag.com/database-performance-tuning/indexing-dos-and-don-ts